.github/workflows/build_and_deploy.yml

```diff
@@ -530,7 +530,7 @@ jobs:
           path: crates/wasm
 
       - name: Create tarballs
-        run: GH_PR_NUMBER=${{ github.event.pull_request && github.event.pull_request.number || '' }} node scripts/create-preview-tarballs.js "${{ github.sha }}" "${{ runner.temp }}/preview-tarballs"
```

Comment:
If we don't use the PR number we aren't able to look up the artifacts currently for the build if it's from a fork. Ideally we could reference PR and commit if available so it can be consistent and we can look it up for a fork

Reply:
That needs separate consideration then. I'd need to see preview build from a fork PR to understand the issue. But withing, preview builds need to be consistent. They cannot reference PRs since that makes them not reproducible.